### PR TITLE
Switching keep card header to Meriweather

### PR DESCRIPTION
**Pull request by andrewconner**
@yipingliao I cleaned up this a bit, it was using `::after`, which wasn't what we wanted all the time.

Feel free to use this font anywhere else.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/app.styl [hunk 1]
```diff
@@ -96,3 +96,16 @@ a[href]
   padding: 2px 0 0
   border-bottom: separatorBorder
   margin: 10px 0
+
+
+@font-face
+  font-family: 'Merriweather'
+  font-style: normal
+  font-weight: 400
+  src: local('Merriweather'), url(//djty7jcqog9qu.cloudfront.net/fonts/mw400.woff) format('woff')
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/app.styl:105]: This is our CDN by the way. Cache headers are set to forever.

File: kifi-backend/modules/shoebox/angular/src/keep/keep.styl [hunk 2]
```diff
@@ -272,13 +262,23 @@ keepBorderRadius = 3px
...
+  top: 2px;
+  position: relative
+  padding: 0 3px
 
 .kf-keep-description
   font-size: 16px
   line-height: 23px
   margin-top: 5px
-  font-family: "Georgia", Arial, sans-serif
+  font-family: "Helvetica", Arial, sans-serif
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/keep/keep.styl:281]: @andrewconner I must know, does your heart twinge a little every time we switch away from Georgia?

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/keep/keep.styl:281]: Nope, it's referring to the country, not the state.

**Comment by yipingliao** [comment]
Thanks! And thanks for the clean-up too.

**Pull request merged**